common4j/src/main/com/microsoft/identity/common/java/controllers/ExceptionAdapter.java

```diff
@@ -436,6 +439,18 @@ public static ClientException clientExceptionFromException(@NonNull final Throwa
 
 
         if (e instanceof OutOfMemoryError) {
+            // Log status code and record exception in span
+            final Span currentSpan = SpanExtension.current();
```

Comment:
What span names are capturing this telemetry?

Reply:
This wouldn't be limited to interactive or silent, but whatever span runs into out of memory exception